clock/src/main/kotlin/io/spine/examples/pingh/clock/ClockContext.kt

```diff
@@ -46,9 +48,55 @@ private val actor = UserId.newBuilder()
 
 /**
  * Emits the `TimePassed` event that contains the current time.
+ *
+ * Use only for test.
```

Comment:
There are a lot of better options:

*  "in testing purposes", 
* "for testing",
* "in tests".

JFYI.